scripts/deploy/images/entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
The purpose of this script is simply to execute the action defined in the named `ConfigMap`. Why create an image for this at all when this script can also be defined in a ConfigMap and a generic image already available in the MCR be used to provide the `bash` shell? This would avoid the overhead of publishing the additional image to MCR.

Reply:
If I understand correctly, what you're suggesting is:
1. We keep the script in the config map and copy it to the pod.
2. Use a generic ubuntu image like you mentioned in the DaemonSet.
3. Execute the script from config map by passing it as "args" to the DaemonSet

??

I wasn't familiar with the script. My only worry would have been availability of nsenter, but looks like we are importing the same image in the dockerfile too.
Sounds like a good plan. I will try this out.

Reply:
Worked!!